scripts/core/EXT_Exp_CounterBasedEventPools.rst

```diff
@@ -0,0 +1,56 @@
+<%
+import re
+from templates import helper as th
+%><%
+    OneApi=tags['$OneApi']
+    x=tags['$x']
+    X=x.upper()
+%>
+:orphan:
+
+.. _ZE_experimental_event_pool_counter_based:
+
+=========================
```

Comment:
minor:  `===` wrapper around the header needs to be as long as the length of the header itself? or doesn't matter? just curious if build complains due to this

Reply:
fixed in latest